machine/v1beta1/types_azureprovider.go

```diff
@@ -145,6 +145,12 @@ type AzureMachineProviderSpec struct {
 	// This is useful for debugging software based launch issues.
 	// +optional
 	Diagnostics AzureDiagnostics `json:"diagnostics,omitempty"`
+	// CapacityReservationGroupID specifies the capacity reservation group resource id that should be
+	// used for allocating the virtual machine.
+	// It is optional but may not be changed once set.
```

Comment:
We don't need to add this line in the godoc.

Reply:
removed.